handsontable/src/dataMap/dataMap.js

```diff
@@ -599,7 +599,7 @@ class DataMap {
    * @param {number} deleteCount Number of rows to remove.
    * @param {...object} elements Row elements to be added.
```

Comment:
I guess you forgot to update the JSDoc after changing the type of the argument

Reply:
Thank you, fixed within f960d969a131e2fbf5d20190b089dd08eefa0d9a.